tests/queries/0_stateless/02353_ascii.sql

```diff
@@ -0,0 +1,2 @@
+SELECT ascii('234');
```

Comment:
Let's add some cases with non-constant column
```suggestion
SELECT ascii('234');
SELECT ascii(materialize('234'));
SELECT ascii(toString(number) || 'abc') from numbers(10);
```